packages/client/src/runtime/core/errors/PrismaClientRustError.ts

```diff
@@ -18,10 +20,12 @@ export class PrismaClientRustError extends Error {
   }
 
   get [Symbol.toStringTag]() {
-    return 'PrismaClientRustPanicError'
+    return 'PrismaClientRustError'
```

Comment:
Found a bug? 😄

Reply:
This might actually explain a few confusing issues we had in the past...

Reply:
Yep, copy-paste from a sibling file, most likely.